crates/iceberg-catalog/Cargo.toml

```diff
@@ -67,6 +67,7 @@ tower-http = { workspace = true, optional = true, features = [
     "util",
 ] }
 tracing = { workspace = true }
+tracing-subscriber = { workspace = true }
```

Comment:
let's move this down to dev-dependencies, i don't think it's needed for the crate itself

Reply:
Done